### PR TITLE
Update Chrome driver v115 onwards with new strategy

### PR DESCRIPTION
**Pull request by Carolina0802**
- Made changes to the *SEResourcesInstaller* script for the Chromer driver copy procedure. Now the variable *$ChromeDrv* is added to the path.

### CODE REVIEW

File: src/SEResourcesInstaller.ps1 [hunk 1]
```diff
@@ -119,23 +119,26 @@ $downloaderWorker = { param([String]$driverOutput, [String]$componentName, [Stri
       function DownloadChromePlatform() {
         if ($platform -eq "WIN") {
           $systemChr = "chromedriver-win32" 
+          $ChromeDrv = "chromedriver.exe"
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1]: Please change this var name `ChromeDrv `for `chromeDrv`

**Review by dloperab — CHANGES_REQUESTED** [review]
Please review the requested changes

File: src/SEResourcesInstaller.ps1 (hunk 1 above)
Comment (Carolina0802) [reply src/SEResourcesInstaller.ps1]: Corrected

**Review by dloperab — APPROVED** [review]
LGTM @Carolina0802. 
Thanks for the work!

**Pull request merged**